diff --git a/src/routes/(sidebar)/+layout.svelte b/src/routes/(sidebar)/+layout.svelte
index 6e3b7c0..05a2d6c 100644
--- a/src/routes/(sidebar)/+layout.svelte
+++ b/src/routes/(sidebar)/+layout.svelte
@@ -2,15 +2,16 @@
 	import '../../app.pcss';
 	import Navbar from './Navbar.svelte';
 	import Sidebar from './Sidebar.svelte';
+	let drawerHidden = false;
 </script>
 
 <header
 	class="sticky top-0 z-40 mx-auto w-full flex-none border-b border-gray-200 bg-white dark:border-gray-600 dark:bg-gray-800"
 >
-	<Navbar />
+	<Navbar bind:drawerHidden />
 </header>
 <div class="overflow-hidden lg:flex">
-	<Sidebar />
+	<Sidebar bind:drawerHidden />
 	<div class="relative h-full w-full overflow-y-auto lg:ml-64">
 		<slot />
 	</div>
diff --git a/src/routes/(sidebar)/Navbar.svelte b/src/routes/(sidebar)/Navbar.svelte
index 4f2d5e7..ccb0a46 100644
--- a/src/routes/(sidebar)/Navbar.svelte
+++ b/src/routes/(sidebar)/Navbar.svelte
@@ -1,45 +1,61 @@
 <script>
 	import AppsMenu from '$lib/widgets/AppsMenu.svelte';
 	import UserMenu from '$lib/widgets/UserMenu.svelte';
-	import { DarkMode, NavBrand, NavHamburger, Navbar, Search, ToolbarButton } from 'flowbite-svelte';
+	import {
+		DarkMode,
+		NavBrand,
+		NavHamburger,
+		NavLi,
+		NavUl,
+		Navbar,
+		Search,
+		ToolbarButton
+	} from 'flowbite-svelte';
 	import { BellSolid } from 'flowbite-svelte-icons';
 	import '../../app.pcss';
 	import Users from '../data/users.json';
 
 	export let fluid = true;
+	export let drawerHidden = false;
 </script>
 
-<Navbar {fluid} color="default">
-	<div class="flex w-fit items-center justify-start">
-		<NavHamburger />
-		<NavBrand href="/" class="ms-2 md:me-24">
-			<img
-				src="/images/flowbite-svelte-icon-logo.svg"
-				class="me-3 h-6 sm:h-8"
-				alt="Flowbite Logo"
-			/>
-			<span class="self-center whitespace-nowrap text-xl font-semibold dark:text-white sm:text-2xl">
-				Flowbite
-			</span>
-		</NavBrand>
-		<div class="hidden lg:block lg:pl-3.5">
-			<Search size="md" class="w-96 border" />
-		</div>
-	</div>
-
-	<div class="ms-auto flex items-center gap-2 p-1">
-		<a
-			class="github-button"
-			href="https://github.com/themesberg/flowbite-svelte-admin-dashboard"
-			data-color-scheme="no-preference: dark; light: light; dark: light;"
-			data-icon="octicon-star"
-			data-size="large"
-			data-show-count="true"
-			aria-label="Star themesberg/flowbite-svelte-admin-dashboard on GitHub">Star</a
+<Navbar {fluid} class="mt-px" color="default">
+	<NavHamburger
+		onClick={() => (drawerHidden = !drawerHidden)}
+		class="m-0 me-3 md:block lg:hidden"
+	/>
+	<NavBrand href="/" class="py-1.5 lg:w-60">
+		<img src="/images/flowbite-svelte-icon-logo.svg" class="me-3 h-6 sm:h-9" alt="Flowbite Logo" />
+		<span class="self-center whitespace-nowrap text-xl font-semibold dark:text-white">Flowbite</span
 		>
+	</NavBrand>
+	<div class="hidden lg:block lg:ps-4">
+		<Search size="md" class="w-96 border" />
+	</div>
+	<div class="ms-auto flex items-center sm:order-2">
 		<ToolbarButton size="lg"><BellSolid /></ToolbarButton>
 		<AppsMenu />
 		<DarkMode />
 		<UserMenu {...Users[4]} />
 	</div>
 </Navbar>
+<!--
+	<Navbar {fluid} color="default">
+	<Search size="md" class="w-96 border" />
+	<NavBrand href="/" class="ms-2 md:me-24">
+		<img src="/images/flowbite-svelte-icon-logo.svg" class="me-3 h-6 sm:h-8" alt="Flowbite Logo" />
+		<span class="self-center whitespace-nowrap text-xl font-semibold dark:text-white sm:text-2xl">
+			Flowbite
+		</span>
+
+	</NavBrand>
+	<NavHamburger />
+
+	<NavUl class="ms-auto flex items-center ">
+		<NavLi>
+			
+		</NavLi>
+
+	</NavUl>
+</Navbar>
+-->
diff --git a/src/routes/(sidebar)/Sidebar.svelte b/src/routes/(sidebar)/Sidebar.svelte
index 9778081..4cea280 100644
--- a/src/routes/(sidebar)/Sidebar.svelte
+++ b/src/routes/(sidebar)/Sidebar.svelte
@@ -24,7 +24,7 @@
 		TableColumnSolid
 	} from 'flowbite-svelte-icons';
 
-	let drawerHidden: boolean = false;
+	export let drawerHidden: boolean = false;
 
 	const closeDrawer = () => {
 		drawerHidden = true;
@@ -111,10 +111,11 @@
 	{nonActiveClass}
 	{activeClass}
 	activeUrl={mainSidebarUrl}
-	asideClass="fixed inset-0 z-30 flex-none h-full w-64 lg:h-auto border-e border-gray-200 dark:border-gray-600 lg:overflow-y-visible lg:pt-20 lg:-mt-2 lg:block"
+	asideClass="fixed inset-0 z-30 flex-none h-full w-64 lg:h-auto border-e border-gray-200 dark:border-gray-600 lg:overflow-y-visible lg:pt-16 lg:block"
 >
+	<h4 class="sr-only">Main menu</h4>
 	<SidebarWrapper
-		divClass="overflow-y-auto px-4 pt-20 lg:pt-4 h-full bg-white scrolling-touch max-w-2xs lg:h-[calc(100vh-4.5rem)] lg:block dark:bg-gray-800 lg:me-0 lg:sticky top-2"
+		divClass="overflow-y-auto px-3 pt-20 lg:pt-7 h-full bg-white scrolling-touch max-w-2xs lg:h-[calc(100vh-4rem)] lg:block dark:bg-gray-800 lg:me-0 lg:sticky top-2"
 	>
 		<nav class="divide-y text-base font-medium">
 			<SidebarGroup ulClass="list-unstyled fw-normal small mb-4 space-y-2">
@@ -132,7 +133,7 @@
 						>
 							<AngleDownOutline slot="arrowdown" class="ms-auto text-gray-800 dark:text-white" />
 							<AngleUpOutline slot="arrowup" class="ms-auto text-gray-800 dark:text-white" />
-							<svelte:component this={icon} slot="icon" />
+							<svelte:component this={icon} slot="icon" class="-me-0.5 h-6 w-6" />
 							{#each Object.entries(children) as [title, href]}
 								<SidebarItem
 									label={title}
@@ -145,7 +146,7 @@
 						</SidebarDropdownWrapper>
 					{:else}
 						<SidebarItem label={name} {href}>
-							<svelte:component this={icon} slot="icon" />
+							<svelte:component this={icon} slot="icon" class="h-6 w-6" />
 						</SidebarItem>
 					{/if}
 				{/each}
diff --git a/src/routes/(sidebar)/dashboard/GeneralInfo.svelte b/src/routes/(sidebar)/dashboard/GeneralInfo.svelte
index 0207b02..bd31230 100644
--- a/src/routes/(sidebar)/dashboard/GeneralInfo.svelte
+++ b/src/routes/(sidebar)/dashboard/GeneralInfo.svelte
@@ -18,9 +18,9 @@
 </script>
 
 <Card size="xl">
-	<Heading tag="h3" class="mb-4 text-xl font-medium tracking-wide dark:text-white"
-		>General Information</Heading
-	>
+	<Heading tag="h3" class="mb-4 text-xl font-medium tracking-wide dark:text-white">
+		General Information
+	</Heading>
 	<form class="grid grid-cols-6 gap-6">
 		{#each inputs as { label, type, placeholder }}
 			<Label class="col-span-6 space-y-2 sm:col-span-3">
@@ -28,6 +28,6 @@
 				<Input {type} {placeholder} class="border font-normal" />
 			</Label>
 		{/each}
-		<Button class="px-2">Save all</Button>
+		<Button class="w-fit whitespace-nowrap">Save all</Button>
 	</form>
 </Card>
